### ISSUE TITLE
Code for downstream detection (COCO) and segmentation (ADE20K) experiments

### ISSUE THREAD

**Issue by showstarpro** [OliverRensu/ARM#10]
Hi, thanks for the great work and for open-sourcing the pretraining/finetuning code!

I noticed the paper reports downstream results on object detection and instance segmentation with Mask R-CNN on COCO (1024×1024), and semantic segmentation with UperNet on ADE20K (512×512), but I couldn't find the corresponding code in this repo.

Would it be possible to release the code/configs for these two downstream experiments, or point me to the framework/config you used (e.g. based on mmdetection / mmsegmentation) so I can reproduce them? We'd like to evaluate a related Vision Mamba pretraining method on the same downstream setup for a fair comparison.

Thanks a lot!
